### PR TITLE
Add recipe for sherlockml-boltzmannclean

### PR DESCRIPTION
**Pull request by shwinnn**
Adds a recipe for the for the sherlockml-boltzmannclean package available on pypi.

### CODE REVIEW

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: Could you change this to use `pip` which is the current convention?

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

And then remove `setuptools` from the `build` requirement and add `pip`

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: As per the comment above, replace `setuptools` with `pip`.

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - numpy
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: It does not appear as if `numpy` is required to actually build the package. If this is the case, please remove it from the `build` requirement list.

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: It looks like this recipe might be a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  home: https://sherlockml.com
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Fills missing values in a pandas DataFrame using a Restricted Boltzmann Machine.'
+  description: |
+    Fills missing values in a pandas DataFrame using a Restricted Boltzmann Machine.
+
+extra:
+  recipe-maintainers:
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: Please confirm that all of the co-maintainers have agreed to be listed here. If you have not spoken to them in a different forum (either online or in person), please ping them on this PR asking for their permission.

File: recipes/sherlockml-boltzmannclean/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - boltzmannclean
+
+about:
+  home: https://sherlockml.com
+  license: Apache 2.0
```
Comment (synapticarbors) [inline recipes/sherlockml-boltzmannclean/meta.yaml]: Change to `Apache-2.0` (we use the conventions listed in https://opensource.org/licenses/alphabetical)

File: recipes/sherlockml-boltzmannclean/meta.yaml (hunk 5 above)
Comment (shwinnn) [reply recipes/sherlockml-boltzmannclean/meta.yaml]: confirmed, (I'm sat with them)

Comment (synapticarbors) [reply recipes/sherlockml-boltzmannclean/meta.yaml]: Great. I'll merge as soon as the CI finishes assuming all of the tests pass.

**Comment by synapticarbors** [comment]
@shwinnn -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**